### PR TITLE
Multiple fixes for airflow docs ingest

### PR DESCRIPTION
**Pull request by mpgreg**
https://github.com/astronomer/ask-astro/issues/111
https://github.com/astronomer/ask-astro/issues/110
https://github.com/astronomer/ask-astro/issues/109
https://github.com/astronomer/ask-astro/issues/108
https://github.com/astronomer/ask-astro/issues/107
https://github.com/astronomer/ask-astro/issues/106
https://github.com/astronomer/ask-astro/issues/105

### CODE REVIEW

**Review by pankajastro — APPROVED** [review]
LGTM, I feel we should test this once in Asto just to avoid any error

File: airflow/include/tasks/split.py
Comment (sunank200) [inline airflow/include/tasks/split.py:102]: Why is this commented on? Maybe we should remove this.

**Review by sunank200 — APPROVED** [review]

**Review by sunank200 — CHANGES_REQUESTED** [review]
Can you update the GitHub PR description for this, please?

I had two questions:
- How does this change affect retrieval?
- @vatsrahul1001 needs to test this change and validate that its improving the related documents before we merge this PR. Just blocking this PR until that is done.

File: airflow/include/tasks/split.py
Comment (mpgreg) [reply airflow/include/tasks/split.py]: Good catch.  Removed in https://github.com/astronomer/ask-astro/pull/112/commits/41aa2f90920cc0983592586dd6666e854136501e

**Pull request closed without merging**
